app/views/layouts/application.html.erb

```diff
@@ -21,11 +21,11 @@
 
   <body>
     <div id="wrapper">
+      <%= render partial: 'govuk_component/breadcrumbs', locals: @breadcrumbs %>
       <main role="main" id="content" class="calculator" lang="<%= I18n.locale %>">
         <% if @education_navigation_ab_test.should_present_new_navigation_view? %>
```

Comment:
I have a concern about this - does this mean the breadcrumb now would appear before the beta banner? Is that likely to happen and how would that look?